server/server.js

```diff
@@ -0,0 +1,31 @@
+const express = require("express");
+const path = require("path");
+
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
+app.use(express.static(path.resolve(__dirname + "../src")));
+app.use("/build", express.static(path.join(__dirname + "../build")));
+
+app.get("/", (req, res) => res.sendFile(path.join(__dirname, "../index.html")));
```

Comment:
Just me not knowing, but what's the distinction between `express.static` on line 11 and `res.sendFile` on 13? I understand the actual difference between those methods, but is there a reason to prefer the latter on 13 over the former?

Reply:
Now that I think about it, it is redundant to have two ways to serve up static files. Line 13 has been removed.